tests/example_dags/.airflowignore

```diff
@@ -0,0 +1,2 @@
+utils.py
+test_example_dags.py
```

Comment:
How hard would it be to move these files outside the `example_dags` folder? It would be cleaner to have a directory only with example DAGs.

Reply:
yes, I agree that is cleaner. Moved them out.